docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -24,12 +24,12 @@ connector:
     - ``null``
   * - ``case-insensitive-name-matching.refresh-period``
```

Comment:
The property name is wrong. It should be `case-insensitive-name-matching.config-file.refresh-period`.